lib/MusicBrainz/Server/Data/Edit.pm

```diff
@@ -112,13 +112,12 @@ sub get_by_id_and_lock
     my ($self, $id) = @_;
 
     my $query =
-        "SELECT " . $self->_columns . " FROM " . $self->_table . " " .
+        "SELECT id FROM edit " .
         "WHERE id = ? FOR UPDATE NOWAIT";
```

Comment:
This is now only for locking the row (without messing with the `edit_data` table). The row is retrieved again below.